pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -72,6 +72,33 @@ const (
 	uninstallHCOErrorMsg  = "The uninstall request failed on dependent components, please check their logs."
 )
 
+var conditions = []conditionsv1.Condition{
```

Comment:
Naming: please consider clearer name for the `conditions` slice. What about `expectedConditions` or something like that?

Reply:
done